tests/suites/test_suite_psa_crypto.data

```diff
@@ -1122,6 +1126,10 @@ PSA cipher: bad order function calls
 depends_on:MBEDTLS_AES_C:MBEDTLS_CIPHER_MODE_CBC
 cipher_bad_order:
 
+PSA symmetric encrypt: AES-ECB, 16 bytes, good
+depends_on:MBEDTLS_AES_C
+cipher_encrypt:PSA_ALG_ECB_NO_PADDING:PSA_KEY_TYPE_AES:"2b7e151628aed2a6abf7158809cf4f3c":"":"6bc1bee22e409f96e93d7e117393172a":"3ad77bb40d7a3660a89ecaf32466ef97":PSA_SUCCESS
+
```

Comment:
3DES isn't in scope for this PR?

Reply:
This PR makes Mbed TLS recognize `PSA_ALG_ECB_NO_PADDING`, so it should test the algorithm with at least a selection of block ciphers. 3DES and other block ciphers should either work correctly or return `NOT_SUPPORTED`. I requested a test for 3DES and not for e.g. CAMELLIA because CAMELLIA is very likely to work if AES does, whereas DES and 3DES could be incorrect due to a misuse of a hard-coded 16 instead of the block size somewhere.